rust/libnewsboat/Cargo.toml

```diff
@@ -46,3 +46,4 @@ features = [ "gettext-system" ]
 tempfile = "3"
 proptest = "0.7"
 section_testing = "0.0.4"
+tempdir = "0.3.7"
```

Comment:
According to [the crates.io](https://crates.io/crates/tempdir), "The tempdir crate is being merged into tempfile". We are already using `tempfile` in tests, even [here in utils](https://github.com/newsboat/newsboat/blob/c01acf47142637b8959036585bc30eaf28c30052/rust/libnewsboat/src/utils.rs#L816-L834). Please revert the changes to Cargo.lock and Cargo.toml, and use `tempfile` instead.

Reply:
Ah! Reminder to myself to check crates.io next time. I went straight to the docs.